README.md

```diff
@@ -28,14 +29,39 @@ or if u wanna use manual mode, enter only command below
 
 ```bash
 create-aio-app
-``` 
+```
 
 After that it will create new directory `my_project`.
 
 ```bash
 cd my_project
 
 make run # start your project
+
+Commands
+
+    Common
+
+run - run the application
+stop - stop the application
+bash - run commands under `bash` in docker container
+test - run tests
+help - shows help
+clean - remove service from the container
+doc - generate html documentation
+
+    Linters
+
+black - formats the code with black
+mypy - checks types with mypy
+lint - formats the code with flake8
+
+    Database
+
+psql - run the application
+migrations - stop the application
+migrate - run tests
+adev - shows help
```

Comment:
we have this description in https://create-aio-app.readthedocs.io/pages/commands.html. do u think that  it'll help to users? Also, this output user will see after created app in console.

Reply:
I agreed that it could not be necessary, so, instead, I just added a link into the `make commands docs`, even though it might seem repetitive, since at the bottom there's already a link to the `docs`, I think it'd help the users to a quicker start. What do u think?